internal/node/hybrid/hybrid.go

```diff
@@ -132,3 +139,40 @@ func (hnp *HybridNodeProvider) getCluster(ctx context.Context) (*types.Cluster,
 
 	return cluster, nil
 }
+
+// UpdateKubeletErrorMessage returns kubelet-specific remediation messages
+func UpdateKubeletErrorMessage(certPath string, err *ValidationError) {
+	if err == nil {
+		return
+	}
+
+	switch err.validationErrorType {
```

Comment:
this switch makes me think what we want here is to use different types for the different errors